src/test/java/de/mediathekview/mserver/crawler/wdr/tasks/WdrFilmDetailTaskTestData.java

```diff
@@ -0,0 +1,123 @@
+package de.mediathekview.mserver.crawler.wdr.tasks;
+
+import de.mediathekview.mlib.daten.GeoLocations;
+import java.time.Duration;
+import java.time.LocalDateTime;
+
+class WdrFilmDetailTaskTestData {
+  private String requestUrl;
+  private String filmPageFile;
+  private String jsUrl;
+  private String jsFile;
+  private String m3u8Url;
+  private String m3u8File;
+  private String topic;
+  private String expectedTitle;
+  private LocalDateTime expectedDate;
+  private Duration expectedDuration;
+  private String expectedDescription;
+  private String expectedSubtitle;
+  private String expectedUrlSmall;
+  private String expectedUrlNormal;
+  private String expectedUrlHd;
+  private GeoLocations[] expectedGeoLocations;
+
+  public WdrFilmDetailTaskTestData(
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Constructor has 16 parameters, which is greater than 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00107)